packages/grid/docker-compose.yml

```diff
@@ -239,6 +237,24 @@ services:
       - "traefik.http.middlewares.${STACK_NAME?Variable not set}-vpn.stripprefix.forceslash=true"
       - "traefik.http.services.${STACK_NAME?Variable not set}-vpn.loadbalancer.server.port=8080"
 
+  weed:
+    profiles:
+      - blob-storage
+    image: "${DOCKER_IMAGE_SEAWEEDFS?Variable not set}"
+    command: "server -s3 -s3.config=/etc/s3config.json -master.volumeSizeLimitMB=1024"
```

Comment:
Q:Any specific reason we limit master server to 1024 MB?
or it because it diverts all storage to volume server, we could keep a low size for master server

Reply:
This option actually makes the master server stop directing writes to oversized volume servers (In this case >024MB/1GB). It helps to keep the volume servers balanced. The value itself (1GB) was arbitrarily chosen from what I know. But we can do performance tests to check what would be the best value.